README.md

```diff
@@ -251,15 +228,27 @@ For launch `foggerid` can be obtained from the miner's account:
 
 Choose `nodeid` for for future identification of this machine. It should be unique alphanumeric value without spaces.
 
-Fog Agent will collect the information about your node and register it in your account
+Starting with version 0.7.0, FR Node is part of Fog Agent. Now Faceter FOG Client is a single component. 
+Fog Agent is installing, running and updating FR Node.
+
+Each action of Fog Agent is written to the terminal and to the log files located along the path `/var/log/fogagent/{Date}.log`, where {Date} is the date the logs were written in the format `20200420`. 
+If Fog Agent is running as a service, then the logs are written only to the log files.
+
+At the first start, Fog Agent will check for the installed Nvidia driver with CUDA and the installed Docker. In case of a positive or negative outcome, it will be written to the logs.
+
+If FR Node has never been launched in the host operating system, then at the first start, Fog Agent will download, install and run all components of FR Node. 
```

Comment:
When environment conditions will be passed, Fog Agent will check Faceter Fog Client docker images existing, their tags then download and install the newest if needed.